examples/imports/defaultWorkbook.csv

```diff
@@ -0,0 +1,51 @@
+firstName,lastName,email
```

Comment:
Can we call this `myWorkbookExample.csv` to match the name of the workbook it's associated with (`MyWorkbook`)?

Reply:
updated